cobrand_hackney/static/app.scss

```diff
@@ -236,3 +236,29 @@ ul.case-list-user li {
         padding: 0 0 0 5px;
     }
 }
+
+// Merging menu
+.lbh-merging-bar {
+    background-color: #00664f47;
+    padding: 1em 0;
+    margin-top:-2.25em;
+    // margin-right: -1em;
+    // margin-left: -1em;
+    border: 1px solid #00664e5d;
```

Comment:
As mentioned in a comment above, this would be easier to read, and more maintainable, if it were:

```scss
border: 1px solid rgba( lbh-color("lbh-a01"), 0.365);
```